layer/layer_data.h

```diff
@@ -240,6 +240,18 @@ class LayerData {
     return proc_addr;
   }
 
+  // Removes a previously created shader module from the LayerData
```

Comment:
nit: missing `.` at the end. Also, could you re-flow the comment? IE put it all one a single line and let clang-format figure out how to break it to fit the column limit.